src/contracts/GPv2AccessControl.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: LGPL-3.0-or-newer
+pragma solidity ^0.6.12;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/EnumerableSet.sol";
+import "./interfaces/GPv2Authentication.sol";
+
+/// @title Gnosis Protocol v2 Access Control Contract
+/// @author Gnosis Developers
+contract GPv2AccessControl is Ownable, GPv2Authentication {
```

Comment:
From what I understand, this should probably be implemented in a way that it is upgradable.

Reply:
Yes, good thinking, this is something I will have to look into. Could this, perhaps, be something we implement independently or do you think it fits well here?

Reply:
We can do this in a separate PR.

Reply:
#167